.travis.yml

```diff
@@ -22,6 +22,7 @@ install:
 
 # command to run tests
 script:
+  - git --version
```

Comment:
Well, there was a problem with Travis' git version: it fails with `git pull --depth 2 --no-edit remote ref`, but works with `git pull --no-edit --depth 2 remote ref`. I had to add the line to see if that was the bug because local tests worked.

I guess it's good to keep that line as reference, just in case for future Travis vs localhost git version conflicts.